frontend/src/components/Step5.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button } from "./Button.tsx";
+import styles from "./Step5.module.css";
+
+// Pass in Application's next function
+export type StepProps = {
+  next: () => void;
+};
+
+export const Step5: React.FC<StepProps> = ({ next }: StepProps) => {
+  // Checks if all Required Inputs are filled before moving on to next step
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const form = event.target as HTMLFormElement;
+    if (!form.checkValidity()) {
+      return;
+    }
+    next();
+  };
+
+  return (
+    <>
+      <form id="step5-form" onSubmit={onSubmit}>
```

Comment:
Missing the border/shaow between the element and the background. 
![image](https://github.com/TritonSE/CCIDC-Application-Overhaul/assets/59126064/58a856f9-f0cc-4cd2-86e4-f7fa496b3be7)


See how there is a shadow/border on the top component but there isn't one on the bottom one